package.json

```diff
@@ -21,32 +21,32 @@
     "release:version": "./scripts/release/version.js"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.11",
-    "@babel/preset-typescript": "^7.12.7",
+    "@babel/core": "^7.12.17",
```

Comment:
All the babel-related updates should be reverted for now. The latest version of Babel breaks the transform.
@ekashida is actively working on this.